File: app/src/main/java/kiwi/slang/MainActivity.java
package kiwi.slang;

import android.graphics.Color;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.LinearLayout;
import android.widget.RelativeLayout;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    private Slangs getOther = new Slangs();
    private TextView btnChangeMsg, randomMsg;
    private RelativeLayout backgroundColorAnimation;
    private ColorsBackground colorsBackground = new ColorsBackground();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView title = findViewById(R.id.title);
        randomMsg = findViewById(R.id.msg);
        btnChangeMsg = (Button) findViewById(R.id.button);
        backgroundColorAnimation = (RelativeLayout) findViewById(R.id.background);

        btnChangeMsg.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                String getClass = getOther.getSlangs();
                randomMsg.setText(getClass);
                int colors = colorsBackground.getColor();
                backgroundColorAnimation.setBackgroundColor(colors);
                btnChangeMsg.setTextColor(colors);
            }
        });
    }
}

File: app/src/main/java/kiwi/slang/MainActivity.kt
package kiwi.slang

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log
import android.widget.Button
import android.widget.RelativeLayout
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    private val getOther = Slangs()
    private lateinit var btnChangeMsg: Button
    private lateinit var randomMsg: TextView
    private lateinit var backgroundColorAnimation: RelativeLayout
    private val colorsBackground = ColorsBackground(this)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.slang_layout)
        Log.v(TAG, "Activity created!")

        randomMsg = findViewById(R.id.msg)
        btnChangeMsg = findViewById(R.id.button)
        backgroundColorAnimation = findViewById(R.id.background)

        btnChangeMsg.setOnClickListener {
            val slang = getOther.getSlangs()
            randomMsg.text = slang
            val color = colorsBackground.color
            backgroundColorAnimation.setBackgroundColor(color)
            btnChangeMsg.setTextColor(color)
        }
    }

    companion object {
        const val TAG = "MainActivity"
    }
}
